lib/img/aws.js

```diff
@@ -45,6 +45,29 @@ AwsDriver.prototype = {
         }
       });
     });
+  },
+
+  delete: function awsDelete(key) {
```

Comment:
huh? The key could be anything, especially with multiple image sizes, there will be suffixes added.

The concern was here?